### PR TITLE
Improve README of Select.

### PR DESCRIPTION
**Pull request by LucasIcarus**
Add 2 samples for README of Select. For #350 

Changes you made in this pull request:

- origin mode with default option
- tags mode with default options

### CODE REVIEW

File: site/yarn.lock [hunk 1]
```diff
@@ -1734,11 +1734,7 @@ domain-browser@^1.1.1:
   version "1.1.7"
   resolved "https://registry.yarnpkg.com/domain-browser/-/domain-browser-1.1.7.tgz#867aa4b093faa05f1de08c06f4d7b21fdf8698bc"
 
-domelementtype@1:
-  version "1.3.0"
-  resolved "https://registry.yarnpkg.com/domelementtype/-/domelementtype-1.3.0.tgz#b17aed82e8ab59e52dd9c19b1756e0fc187204c2"
-
-domelementtype@~1.1.1:
+domelementtype@1, domelementtype@~1.1.1:
```
Comment (cpylua) [inline site/yarn.lock]: revert 会去，没改啥

File: packages/zent/src/select/README.md [hunk 2]
```diff
@@ -272,12 +289,35 @@ ReactDOM.render(
 ```
 :::
 
+:::demo 支持多选默认选项
```
Comment (cpylua) [inline packages/zent/src/select/README.md]: 看不懂。。。

File: packages/zent/src/select/README.md [hunk 3]
```diff
@@ -272,12 +289,35 @@ ReactDOM.render(
...
+```
+:::
+
 ### API
 
 | 参数 | 说明 | 类型 | 默认值 | 是否必填 |
 |------|------|------|--------|--------|
 | data | 选项数据 | array | `[]` | 是 |
-| value | 选中的值，当为tags类型时，可以传入数组 | any | `''` | 否 |
+| value | 默认选中的值，当为tags类型时，可以传入数组 | any | `''` | 否 |
```
Comment (cpylua) [inline packages/zent/src/select/README.md]: 为什么加个默认？

**Comment by LucasIcarus** [comment]
value and index prop of Select is designed to make it as a controlled component.

through them to set up the default option is really bad idea and needs hack in actual using.

是我太年轻了。。

**Pull request closed without merging**